scripts/pinned-tools.sh

```diff
@@ -104,13 +104,28 @@ function setAssemblyName() {
 
   if [ "$#" -ne 1 ]; then
     echo "'setAssemblyName' expects only one argument."
-    echo "Example usage: 'setAssemblyName my_prefix'"
+    echo "Example usage: 'setAssemblyName <assembly prefix>'"
     exit 1
   fi
 
   ASSEMBLY_NAME="${1}_${GIT_HASH}"
 }
 
+# Uploads an assembly, given an assembly prefix and a project name
+function uploadAssembly() {
+  if [ "$#" -ne 2 ]; then
+    echo "'uploadAssembly' expects two arguments."
+    echo "Example usage: 'uploadAssembly <assembly prefix> <project name>'"
```

Comment:
Do we need to support the project name argument?

Reply:
yes, this is to defend against the case where the `spatialos.json` refers to a different project than the deployment launcher defaults. controlling the project end-to-end gives us guarantee that if this step succeeds that the assembly has been uploaded to the project we will deploy in down the line.